### ISSUE TITLE
Test compilation failed due to a typo.

### ISSUE THREAD

**Issue by tsangwpx** [Unidata/netcdf-fortran#111]
In nf_test/nftst_rengrps.F, the 6th column is a letter "i" as continuation line. ifort reported this as an error.

See 
https://github.com/Unidata/netcdf-fortran/blob/41a6e586633ae2fe5b6efcd78a8e5737635adc7e/nf_test/ftst_rengrps.F#L14

The source code is 4.4.5 tarball downloaded from releases.

**Comment by WardF** [comment]
Thanks for the bug report, curious that gfortran doesn't report this as an error. We will check it out!

**Comment by stefan7487** [comment]
FWIW - This seems to be fixed by adding one additional space before the "include" in line 14 of nf_test/nftst_rengrps.F

Adding in that space, I was able to compile and pass all tests of 4.4.5 associated with netCDF-C 4.6.2. This was using the latest version of Intel compilers.

**Comment by WardF** [comment]
That is a very easy fix, thanks.  I will incorporate that fix shortly.

**Comment by olevski** [comment]
Just wanted to say that I came across the same issue when using the PGI compiler and the source code for netcdf-c 4.6.3 and netcdf-f 4.4.5. I guess I should have used the latest source code from the repo instead of grabbing the release here https://www.unidata.ucar.edu/downloads/netcdf/index.jsp. I can also confirm that gfortran worked even before the indentation was fixed. Knowing absolutely no Fortran it took me a while to figure out that the lack of extra space was the issue. For anyone who may be googling the error, this is how it is reported by PGI in the terminal:
`PGF90-S-0034-Syntax error at or near identifier no (ftst_rengrps.F: 13)`
